### PR TITLE
module_utils/ipa.py: fix regex when parsing version

### PR DESCRIPTION
**Pull request by SJay3**
##### SUMMARY

Fix module_utils/ipa.py regex to parsing freeipa version

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

module_utils/ipa.py

##### ADDITIONAL INFORMATION

Right regex for parsing
https://regex101.com/r/s2ecr6/1

### CODE REVIEW

**Comment by felixfontein** [comment]
Completely surprising, two-digit version numbers exist ;-) Thanks for spotting and fixing this! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by SJay3** [comment]
@felixfontein I was suprising too!) 
Added changelog fragments

File: changelogs/fragments/8175-get_ipa_version_regex.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ipa - fix get version regex im ipa module_utils (https://github.com/ansible-collections/community.general/pull/8175)
```
Comment (felixfontein) [inline changelogs/fragments/8175-get_ipa_version_regex.yml:2]:
```suggestion
  - ipa - fix get version regex in IPA module_utils (https://github.com/ansible-collections/community.general/pull/8175).
```

**Comment by felixfontein** [comment]
If nobody objects, I'll merge this in ~a week.

**Comment by felixfontein** [comment]
@SJay3 thanks for fixing this! And thanks to @aBUDmdBQ for creating a very similar PR as well (#8201) :)

**Pull request merged**